app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
404 or 500?

500. Sheesh. Brains :)

Master is doing the "wrong" thing for me. I'll see if I can show you a particular page (for me it was the contributing/canonical-data/* pages).

Maybe not this week, though.